<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogHoughLines</class>
 <widget class="QDialog" name="DialogHoughLines">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0">
   <item row="0" column="0" rowspan="14">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="lblTheta">
     <property name="text">
      <string>theta</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblRho">
     <property name="text">
      <string>rho</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="dblSBRho">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>2000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QRadioButton" name="rBProb">
     <property name="text">
      <string>probabilistic</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDoubleSpinBox" name="dblSBTheta">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lblDivRho">
     <property name="text">
      <string>divisor rho</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lblThreshold">
     <property name="text">
      <string>threshold</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="lblMinLength">
     <property name="text">
      <string>minimal length</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QDoubleSpinBox" name="dblSBMinLength">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>300.000000000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="lblMaxGap">
     <property name="text">
      <string>maximal gap</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="sBThreshold">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="value">
      <number>150</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="dblSBDivRho">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="lblDivTheta">
     <property name="text">
      <string>divisor theta</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDoubleSpinBox" name="dblSBDivTheta">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QDoubleSpinBox" name="dblSBMaxGap">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>2000.000000000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>line thickness</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QSlider" name="sliderLineThickness">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>color</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QSlider" name="sliderColor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="2">
    <widget class="QCheckBox" name="cBDrawLines">
     <property name="text">
      <string>draw lines</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
